src/routes/JobOpportunity/JobOpportunity.tsx

```diff
@@ -137,6 +178,18 @@ const JobOpportunity = () => {
       ),
       showInDevOnly: true,
     },
+    {
+      navTitle: t('more-information'),
+      content: (
+        <div className="grid w-full grow grid-cols-2 gap-6">
+          {Object.keys(jakaumat)
+            .filter((key) => !['osaaminen', 'ammatti'].includes(key))
+            .map((key) => (
+              <JakaumaList key={key} name={key as keyof Jakaumat} />
+            ))}
```

Comment:
By moving the `Keys` out from the **JakaumaList**, it could be reused here:

```tsx
          {(Object.keys(jakaumat) as Keys)
            .filter((key) => !['osaaminen', 'ammatti'].includes(key))
            .map((key) => (
              <JakaumaList key={key} name={key} />
            ))}
```
This also uses the type assertion like the previous implementation, but key is now in correct type the whole time (and not as `string`).